java/zulu-jdk8/Portfile

```diff
@@ -0,0 +1,205 @@
+        sha256                 5ff89e32f8f35a8771fb284f7b6acc5c5c55d8f74c255011d63487ac7a1dd044 \
+        size                   183295140 \
+                               zulu${zulu_version(x64)}-ca-jdk${jdk_version(x64)}-macosx_x64.zip \
+        rmd160                 39c0fbd54d53db9b671a9d49a52ca0d11c0dddb8\
+        sha256                 306e0c365c144752f204fd74df17d56229f23f2e73b858d8813469676c90a038\
+        size                   204873866
+}
+
+subport zulu-jdk14 {
+
+    set jdk_version(x64)       14.0.2
+    set zulu_version(x64)      14.29.23
+    set major                  14
+    revision                   0
+
+    supported_archs            x86_64
+
+    checksums                  zulu${zulu_version(x64)}-ca-jdk${jdk_version(x64)}-macosx_x64.zip \
+        rmd160                 f97f9d7d7ad526797ff8dcd32ab9c5187aa4f5cb \
+        sha256                 57e737d639a6927c82626fd18257751e6bb4d64e291b8fac62ddad01255a1d77 \
+        size                   207255111
+}
+
+subport zulu-jdk15 {
+
+    set jdk_version(x64)       15.0.1
+    set zulu_version(x64)      15.28.51
+    set major                  15
+    revision                   0
+
+    supported_archs            x86_64
+
+    checksums                  zulu${zulu_version(x64)}-ca-jdk${jdk_version(x64)}-macosx_x64.zip \
+        rmd160                 20f1b78a26beab1cd71c2fd99107704df53fc3e7 \
+        sha256                 084c67b6ada87e8ba1f8abf35af2c8a56b204ccf513600af6e09ae7880824a84 \
+        size                   206369908
+}
+
+categories       java devel
+maintainers      nomaintainer
+platforms        darwin
+license          GPL-2
+homepage         https://www.azul.com/downloads/zulu-community/
+
+
+set zulu_arch(arm64)       aarch64
+set zulu_arch(x86_64)      x64
+
+if {![info exists zulu_arch(${build_arch})]} {
+    set zulu_arch(${build_arch}) ${build_arch}
+}
+
+set zulu_name(arm64)       macos
+set zulu_name(x86_64)      macosx
+
+if {![info exists zulu_arch(${build_arch})]} {
+    set zulu_arch(${build_arch}) macosx
+}
+
+if {${os.platform} eq "darwin" && ${os.major} < 14} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} ${version} is only supported on Mac OS X 10.10 Yosemite or later."
+        return -code error
+    }
+}
+
+description  Open Java Development Kit ${major} (Zulu) with HotSpot VM
+long_description OpenJDK build provided by Azul, built from a fully \
+    open-source set of build scripts and infrastructure. \
+    \
+    HotSpot is the VM from the OpenJDK community and  the most widely used VM. \
+    It is suitable for all workloads.
+
+set version  $zulu_version($zulu_arch($build_arch))
+
+master_sites https://cdn.azul.com/zulu/bin/
+distname     zulu$zulu_version($zulu_arch($build_arch))-ca-jdk$jdk_version($zulu_arch($build_arch))-$zulu_name($build_arch)_$zulu_arch($build_arch)
+use_zip      yes
+worksrcdir   zulu$zulu_version($zulu_arch($build_arch))
+configure.cxx_stdlib libstdc++
```

Comment:
Really? libstdc++ has been deprecated ever since OS X 10.9 Mavericks so it would surprise me if software that requires OS X 10.10 Yosemite or later used libstdc++.

Reply:
@ryandesign without it it can't be installed with erorr like
```
zulu-jdk8 is using libstdc++ (this installation is configured to use libc++)
```

Reply:
Let me use it only where it is required.

Reply:
I'm just expressing surprise that anything made recently would be using libstdc++, since it has been deprecated for so many years already. As long as this software does not use any C++ libraries from any other ports, and does not provide any C++ libraries that would be used by other ports, you can force the use of libstdc++.

Reply:
@ryandesign I've used only where it is required. All new build doesn't required it.